File: app-core/src/main/java/ru/micron/persistence/repository/FilmReceiveRepository.java
package ru.micron.persistence.repository;

import java.util.List;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;
import ru.micron.persistence.model.FilmReceive;

@Repository
public interface FilmReceiveRepository extends JpaRepository<FilmReceive, String> {

  List<FilmReceive> findFilmReceivesByIsReceivedFalse();
}

File: app-core/src/main/kotlin/ru/micron/persistence/repository/FilmReceiveRepository.kt
package ru.micron.persistence.repository

import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository
import ru.micron.persistence.model.FilmReceive

@Repository
interface FilmReceiveRepository : JpaRepository<FilmReceive, String> {
    fun findFilmReceivesByIsReceivedFalse(): List<FilmReceive>
}